### ISSUE TITLE
Parameterized TF type?

### ISSUE THREAD

**Issue by bennn** [utahplt/FloatTracker.jl#37]
If we had one type for TrackedFloats, FT would be easier to use and maintain.

Question 1: instead of having N types for TrackedFloat (`TrackedFloat16`, ...), could we use one parameterized type?

Question 2: if it's one type, do we lose anything important --- such as checks that a number never exceeds 32 bits? (Are those checks useful??)

**Comment by dpsanders** [comment]
Is there any progress on this? This seems easy (ish) to implement and would improve flexibility (e.g. tracking of `BigFloat`s would be possible).

**Comment by bennn** [comment]
Nope, no progress. We're looking for someone to take the lead on this project.
